x/nameservice/05_params.md

```diff
@@ -0,0 +1,15 @@
+<!--
+order: 7
+-->
+
+# Parameters
+
+The nameservice module contains the following parameters:
+
+| Key                    | Type            | Example |
+| ---------------------- | --------------- | ------- |
+| MaxMemoCharacters      | string (uint64) | "256"   |
+| TxSigLimit             | string (uint64) | "7"     |
+| TxSizeCostPerByte      | string (uint64) | "10"    |
+| SigVerifyCostED25519   | string (uint64) | "590"   |
+| SigVerifyCostSecp256k1 | string (uint64) | "1000"  |
```

Comment:
```suggestion
|     |     |    |
```

Reply:
I wasn't clear on how to glean parameters, so this set of parameters is just a copy of the evidence module spec. Replace with accurate nameservice parameters.